examples/node-app-mssql/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM node:10
+
+WORKDIR /app/
+
+COPY . .
+
+RUN yarn
+
+CMD ["yarn", "docker-start"]
```

Comment:
Explicitly adding `yarn sync-db && yarn start` would make for sense for the users so that they know first the `sync-db` cli is run to synchronize the database and only then the app is started.

Reply:
Done